NBug/Core/UI/WinForms/Full.cs

```diff
@@ -19,93 +19,105 @@
+            reportContentsTabPage.Text = Settings.Resources.UI_Dialog_Full_Report_Contents_Tab;
+            errorDescriptionLabel.Text = Settings.Resources.UI_Dialog_Full_How_to_Reproduce_the_Error_Notification;
+            quitButton.Text = Settings.Resources.UI_Dialog_Full_Quit_Button;
+            sendAndQuitButton.Text = Settings.Resources.UI_Dialog_Full_Send_and_Quit_Button;
+
+            // ToDo: Displaying report contents properly requires some more work.
+            mainTabs.TabPages.Remove(mainTabs.TabPages["reportContentsTabPage"]);
+        }
+
+        internal UIDialogResult ShowDialog(SerializableException exception, Report report)
+        {
+            Text = string.Format("{0} {1}", report.GeneralInfo.HostApplication, Settings.Resources.UI_Dialog_Full_Title);
+
+            _lastException = exception;
+
+            // Scaling
+            sendAndQuitButton.Image = DpiUtil.Scale(Resources.Send);
+            btnCopy.Image = DpiUtil.Scale(Resources.CopyToClipboard);
+            exceptionTypeLabel.Image = DpiUtil.Scale(Resources.NBug_Icon_PNG_16);
+            exceptionDetails.InformationColumnWidth = DpiUtil.Scale(350);
+            exceptionDetails.PropertyColumnWidth = DpiUtil.Scale(101);
+
+            // Fill in the 'General' tab
+            warningPictureBox.Image = SystemIcons.Warning.ToBitmap();
+            exceptionTextBox.Text = exception.Type;
+            exceptionMessageTextBox.Text = exception.Message;
+            targetSiteTextBox.Text = exception.TargetSite;
+            applicationTextBox.Text = $@"{report.GeneralInfo.HostApplication} [{report.GeneralInfo.HostApplicationVersion}]";
+            nbugTextBox.Text = report.GeneralInfo.NBugVersion;
+            dateTimeTextBox.Text = report.GeneralInfo.DateTime;
+            clrTextBox.Text = report.GeneralInfo.CLRVersion;
+
+            // Fill in the 'Exception' tab
+            exceptionDetails.Initialize(exception);
+
+            // ToDo: Fill in the 'Report Contents' tab);
+            ShowDialog();
+
+            // Write back the user description (as we passed 'report' as a reference since it is a reference object anyway)
+            report.GeneralInfo.UserDescription = descriptionTextBox.Text;
+            return _uiDialogResult;
+        }
+
+        private void QuitButton_Click(object sender, EventArgs e)
+        {
+            _uiDialogResult = new UIDialogResult(ExecutionFlow.BreakExecution, SendReport.DoNotSend);
+            Close();
+        }
+
+        private void SendAndQuitButton_Click(object sender, EventArgs e)
+        {
+            if (DialogResult.No == MessageBox.Show(this,
+                    @"Please give as much as information as possible to help the developers solve this issue. Otherwise, your issue ticket may be closed without any follow-up from the developers.
+
+Because of this, please make sure to fill in all the fields in the report template.
+
+Do you wish to proceed?",

```

Comment:
What do you think about "Send report anyway?" Is it clearer?

Reply:
👍